include/aikido/planner/dart/ConfigurationToConfiguration_to_ConfigurationToTSR.hpp

```diff
@@ -10,12 +10,14 @@ namespace planner {
 namespace dart {
 
 class ConfigurationToConfiguration_to_ConfigurationToTSR
-    : public PlannerAdapter<ConfigurationToConfigurationPlanner,
+    : public PlannerAdapter<aikido::planner::
```

Comment:
Nit: same as @brianhou 's comment above, `aikido::planner` wouldn't be necessary here.